cmd/apiserver/main.go

```
@@ -39,29 +56,28 @@ func run(
 	ctx, cancel := signal.NotifyContext(ctx, os.Interrupt)
 	defer cancel()
 
-	log, err := logger.NewZapLogger()
+	defer logger.Log.Sync() // flushes any buffered log entries
```

golangci-lint (1.62.2)

Error return value of `logger.Log.Sync` is not checked

(errcheck)